tests/epyccel/test_epyccel_functions.py

```diff
@@ -367,6 +367,14 @@ def f22(a):
     f = epyccel(f22, language=language)
     assert f(complex(1, 2.2)) == f22(complex(1, 2.2))
 
+def test_pass(language):
+    @types('int')
+    def f22(a):
+        pass
+        return a
```

Comment:
To resolve this Codacy error, remove `return a`, the code will not reach the return statement because of the keyword pass.

Reply:
i see, so im gonna add a codegen test for the function